resources/interactive.mjs

```diff
@@ -211,41 +199,34 @@ function createUIForSuites(suites, onStep, onRunSuites)
 
 const searchParams = new URLSearchParams(window.location.search);
 
-function disableAllSuitesExcept(suiteName)
-{
+function disableAllSuitesExcept(suiteName) {
     let foundMatching = false;
-    Suites.forEach(element => {
+    Suites.forEach((element) => {
         if (element.name !== suiteName)
             element.disabled = true;
         else
             foundMatching = true;
     });
     if (!foundMatching)
         throw Error(`No matching suite for: "${suiteName}"`);
-
 }
 
-function startTest()
-{
-    // FIXME: Use global params here 
-    if (searchParams.has('suite'))
-        disableAllSuitesExcept(searchParams.get('suite'));
+function startTest() {
+    // FIXME: Use global params here
+    if (searchParams.has("suite"))
+        disableAllSuitesExcept(searchParams.get("suite"));
 
-    const interactiveRunner =  new window.BenchmarkRunner(Suites, params.iterationCount);
+    const interactiveRunner = new window.BenchmarkRunner(Suites, params.iterationCount);
 
     if (!(interactiveRunner instanceof InteractiveBenchmarkRunner)) {
-        throw Error(
-            "window.BenchmarkRunner must be "+
-            "a subclass of InteractiveBenchmarkRunner");
+        throw Error("window.BenchmarkRunner must be " + "a subclass of InteractiveBenchmarkRunner");
```

Comment:
Looks like this can just be a single string literal?

Reply:
nice catch - I can address that with the template literals update in a follow up pr